ui/src/app/base/components/Section/Section.js

```diff
@@ -23,6 +42,7 @@ const Section = ({ children, sidebar, title }) => {
           </Col>
         )}
         <Col size={sidebar ? 10 : 12} className="section__content">
+          {generateMessages(messages, dispatch)}
```

Comment:
This could probably be a separate component - NotificationsList or something.

Reply:
Good idea. Done.